### PR TITLE
Tests: route configuration & context setter

### PR DESCRIPTION
**Pull request by OmAximani0**
## Fixes Issue
Refs: #148 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: pkg/dashboard/api_test.go [hunk 1]
```diff
@@ -64,3 +65,47 @@ func TestConfigureStatic(t *testing.T) {
...
+
+	assert.Equal(t, w.Code, http.StatusOK)
+}
+
+func TestContextSetter(t *testing.T) {
+	w := httptest.NewRecorder()
+	con := GetTestGinContext(w)
+
+	// Required arguements
+	data, _ := objects.NewDataLayer("TestSpace", "T-1", objects.NewHelmConfig)
```
Comment (undera) [inline pkg/dashboard/api_test.go:101]: The error here should be handled to fail the test, not ignored

File: pkg/dashboard/api_test.go [hunk 2]
```diff
@@ -64,3 +65,47 @@ func TestConfigureStatic(t *testing.T) {
...
+
+	// Set the context
+	ctxHandler := contextSetter(data)
+	ctxHandler(con)
+
+	appName, _ := con.Get("app")
```
Comment (undera) [inline pkg/dashboard/api_test.go:107]: Error should be used to fail the test

File: pkg/dashboard/api_test.go (hunk 1 above)
Comment (OmAximani0) [reply pkg/dashboard/api_test.go:101]: Ok sir.
Sir can I use like t.Fatal(err) ?

Comment (undera) [reply pkg/dashboard/api_test.go]: Yes, that's what you should use

**Pull request merged**